src/OpenSSL/crypto.py

```diff
@@ -741,9 +739,7 @@ def __str__(self):
 
         bio = _new_mem_buf()
         print_result = _lib.X509V3_EXT_print(bio, self._extension, 0, 0)
-        if not print_result:
-            # TODO: This is untested.
-            _raise_current_error()
+        _openssl_assert(print_result)
```

Comment:
what does X509V3_EXT_print actually return?  _openssl_assert does an `is` comparison so this check seems shaky.